src/bindings/bnd_extensions.cpp

```diff
@@ -539,15 +539,7 @@ BND_Bitmap* BND_File3dmBitmapTable::FindId(BND_UUID id)
 void BND_File3dmLayerTable::Add(const BND_Layer& layer)
```

Comment:
Thanks for fixing this! Out of interest, why not return `int` (index of newly added layer) here?

Reply:
My goal is to match the RhinoCommon C# API as close as possible in order to do things like autogenerate documentation.
https://developer.rhino3d.com/api/RhinoCommon/html/M_Rhino_FileIO_File3dmCommonComponentTable_1_Add.htm

Reply:
My mistake. I thought that this function did return `int` in RhinoCommon.